### PR TITLE
Add unit tests for appointment logic

### PR DESCRIPTION
**Pull request by nicoleang09**
Also fixed incorrect behaviour of addAppt that was discovered.

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/AddAppointmentCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        //                + DATE_DESC_PTM + CONTACT_DESC_2, new AddAppointmentCommand(expectedAppointment));
+
+        // multiple names - last name accepted
+        //        assertParseSuccess(parser, NAME_DESC_PLAY_DATE + NAME_DESC_PTM + ADDRESS_DESC_PTM
+        //                + DATE_DESC_PTM + CONTACT_DESC_2, new AddAppointmentCommand(expectedAppointment));
+
+        // multiple addresses - last address accepted
+        //        assertParseSuccess(parser, NAME_DESC_PTM + ADDRESS_DESC_PTM + DATE_DESC_PTM
+        //                + CONTACT_DESC_2, new AddAppointmentCommand(expectedAppointment));
+    }
```
Comment (mechastriker3) [inline src/test/java/seedu/address/logic/parser/AddAppointmentCommandParserTest.java:64]: Are these commented out stuff important?

Comment (mechastriker3) [reply src/test/java/seedu/address/logic/parser/AddAppointmentCommandParserTest.java:64]: But i think looks ok overall, let's merge it !!!

**Review by mechastriker3 — COMMENTED** [review]
Looks good!

**Review by Stratostorm — APPROVED** [review]
Looks good to merge

**Review by edelinetenges — APPROVED** [review]
Looks good to merge!

**Pull request merged**